README.md

```diff
@@ -78,3 +78,79 @@ To build the full distribution, with all database bindings:
 To build a single database binding:
 
     mvn -pl site.ycsb:mongodb-binding -am clean package
+
+Running multiple instances and latency percentiles
+--------------------------------------------------
+
+In general, you shall be interested in 99% percentile (P99) of the latency
+distribution and the rest - 99.9%, 99.99%, 99.999% - the latency tail.
+95% percentile has a very low probability to be observed by the user
```

Comment:
>  95% percentile has a very low probability to be observed by the user...

This was a little confusing to me at first.

Maybe reword to something like:

> A user is very unlikely to have _all_ of their requests happen within the 95% percentile, so don't even bother.

Reply:
I changed the wording to

> In general, you shall be interested in 99% percentile (P99) of the latency
distribution, and the rest of the tail - 99.9%, 99.99%, 99.999%. The difference
between the amount of requests that will be observed by a user that fall
into 95% (P95) percentile and 99% percentile may be sufficiently large.

> For example, see "How Many Nines?" at https://bravenewgeek.com/everything-you-know-about-latency-is-wrong/.
The formula to calculate probability of how many clients will observe
a specific percentile is: